src/main/java/reposense/model/StandaloneConfig.java

```diff
@@ -26,7 +28,7 @@
 
     public List<String> getFormats() {
         if (formats == null) {
-            return Collections.emptyList();
+            return ArgsParser.DEFAULT_FORMATS;
```

Comment:
hmm I guess this was the reason why our contribution bar was empty earlier.

Btw I think we should customize our standalone config's formats as well now that its in place.
Perhaps let's add the following to our formats?

.md, .java, .js, .jade, .scss, .gradle, .yml 

Let me know if i missed out any.

Reply:
nvm, leave the _reposense config alone for now.

Reply:
Will use `["gradle", "jade", "java", "js", "md", "scss", "yml"]` in `_reposense/config.json`